lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/aplinear/APLOutlierSummarizer.java

```diff
@@ -24,6 +24,11 @@
         return Arrays.asList("Outliers", "Count");
     }
 
+    @Override
+    public int[][] getEncoded(List<String[]> columns, DataFrame input) {
```

Comment:
Ah, so here you need `input`. Hmmmm………I don't have a good solution for this. I guess we can keep it as is